tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Settings/Scripts.spec.ts

```diff
@@ -0,0 +1,250 @@
+import {ConstantHelper, test} from "@umbraco/playwright-testhelpers";
+import {expect} from "@playwright/test";
+
+test.describe('Script tests', () => {
+
+  const scriptName = 'TestScript';
+  const scriptPath = scriptName + '.js';
+  const scriptFolderName = 'TestScriptFolder';
+
+  test('can create a empty script', async ({page, umbracoApi, umbracoUi}) => {
+    // Arrange
+    await umbracoApi.script.ensureNameNotExists(scriptPath);
+
+    // Act
+    await page.goto(umbracoApi.baseUrl + '/umbraco');
+    await umbracoUi.goToSection(ConstantHelper.sections.settings);
```

Comment:
it always run these piece of code before all tests, so should be put in beforeEach()
     await page.goto(umbracoApi.baseUrl + '/umbraco');
    await umbracoUi.goToSection(ConstantHelper.sections.settings);